spec/features/assigned_articles_spec.rb

```diff
@@ -21,9 +21,9 @@
     # we need to use VCR to avoid getting stopped by WebMock
     VCR.use_cassette('assigned_articles_view') do
       visit "/courses/#{course.slug}/articles/assigned"
-      expect(page).to have_content('Nancy Tuana')
+      page.has_content?('Nancy Tuana')
```

Comment:
This looks like it simply removes the check for the expected article, so that if `.has_content?` returns `false` it doesn't matter and the test continues... whereas the `expect` line also relies on Capybara to wait a certain amount of time and continue checking for the content, so it ensure that the assigned articles list has rendered before moving on. I don't think will make it more reliable.